### PR TITLE
Update instructions.md

### PR DESCRIPTION
**Pull request by chrisguida**
Recommend connecting to Sphinx's node since it results in much better reliability.

### CODE REVIEW

File: docs/instructions.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Using Sphinx requires that you have _both_ inbound and outbound liquidity on you
 
 ### Getting Outbound Liquidity
 
-Getting outbound liquidity is easy; just open a channel with a well-connected node and fund it with 100,000 sats. You can open a channel with Start9 [here](https://1ml.com/node/025d28dc4c4f5ce4194c31c3109129cd741fafc1ff2f6ea53f97de2f58877b2295) and/or you can open a channel with Sphinx's recommended node [here](https://github.com/stakwork/sphinx-relay/wiki/Home-node-FAQ).
+Follow these instructions [these instructions](https://github.com/stakwork/sphinx-relay/wiki/Home-node-FAQ) to open a channel with Sphinx's recommended node in the correct way so that the zero-fee transactions Sphinx makes will work smoothly. You can also open a channel with [Start9's node](https://1ml.com/node/025d28dc4c4f5ce4194c31c3109129cd741fafc1ff2f6ea53f97de2f58877b2295), but Sphinx may not be able to find zero-fee routes as easily.
```
Comment (MattDHill) [inline docs/instructions.md:15]: Duplicate "these instructions"

Comment (MattDHill) [reply docs/instructions.md:15]: Also first sentence is not a complete sentence. Maybe a period after "...Sphinx's recommended node". Then, "This is the correct..."

Comment (chrisguida) [reply docs/instructions.md:15]: Your first point is correct, thanks

Your second point is not correct. It's definitely a complete sentence. You not only need to open to Sphinx's recommended node; you also need to do so in the way that they recommend. I'm open to rewordings, but I don't want this meaning to be lost.

Comment (MattDHill) [reply docs/instructions.md:15]: I read "in the correct way" as "is the correct way".

I think my suggestion above gets the job done.

Comment (islandbitcoin) [reply docs/instructions.md:15]: I will await resolution on the wording before completing my review. the rest of it LTGM

Comment (chrisguida) [reply docs/instructions.md]: Instructions overhauled, please re-review

File: docs/instructions.md [hunk 2]
```diff
@@ -1,39 +1,84 @@
...
+
+The Sphinx Chat client app, when connected to your Embassy Sphinx Relay server,
+offers decentralized podcasting, community channels (Tribes), and DMs. While the
+DM feature is currently a bit slow and clunky, the Tribes and Podcasting
+features work great. In our testing, the first DM was dropped, but after that
+DMs went through eventually. YMMV. Following the
+[official Sphinx setup guide](https://github.com/stakwork/sphinx-relay/wiki/Home-node-FAQ)
+(as indicated above) will help optimize this experience.
+
+# Lightning payments as spam control
```
Comment (MattDHill) [inline docs/instructions.md:66]: This section seems purely informative (not instructions) and out of place. Also has the largest heading!

File: docs/instructions.md [hunk 3]
```diff
@@ -1,39 +1,84 @@
...
+
+The Sphinx Chat protocol uses Lighting micropayments to send messages using the
+lightning network. This allows both DMs and Tribes to protect themselves against
+spammers. These payments are refunded after a while if no spam/abuse is
+detected.
 
 ### Back up your Sphinx Client App!
```
Comment (MattDHill) [inline docs/instructions.md:73]: This section deserves the ## heading to replace the unnecessary section above.

File: docs/instructions.md (hunk 2 above)
Comment (chrisguida) [reply docs/instructions.md:66]: Should we not put this info anywhere? Seems like people should know what the payments are for.

File: docs/instructions.md (hunk 3 above)
Comment (chrisguida) [reply docs/instructions.md:73]: Will do

Comment (chrisguida) [reply docs/instructions.md:73]: Done

File: docs/instructions.md (hunk 2 above)
Comment (chrisguida) [reply docs/instructions.md]: @MattDHill waiting for your response

Comment (chrisguida) [reply docs/instructions.md]: Headings resized btw

Comment (chrisguida) [reply docs/instructions.md]: How about we just move this to the top?

**Pull request merged**